src/Form/RegistrationRecoveryTokenType.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @author doobry <doobry@systemli.org>
+ */
+class RegistrationRecoveryTokenType extends AbstractType
+{
+    const NAME = 'registration_recovery_token';
+
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('ack', CheckboxType::class, [
+                'required' => false,
```

Comment:
This must be `true` otherwise the Checkbox is optional and validation is only made with ConstraintValidator.

Reply:
Actually, that's what I wanted to achieve. I didn't like the HTML5 error layout, so I did it that way to instead display the validation error from Symfony ConstraintValidator.

Reply:
Done now.